.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -10,5 +10,4 @@ assignees: ''
 ** Instructions **
 This issue tracker is intended to report bugs in the examples such as typos, broken links, etc.
 
-The issue tracker should *NOT* be used for asking for help, making feature suggestions, etc. To ask for help or give general feedback please post in the LabVIEW NXG section of the LabVIEW Discussion Forums:
-https://forums.ni.com/t5/LabVIEW/bd-p/170
+The issue tracker should *NOT* be used for asking for help, making feature suggestions, etc. To ask for help or give general feedback please post in the [G Web Development Software Discussion Forums](https://forums.ni.com/t5/G-Web-Development-Software/bd-p/g-web-development?profile.language=en0).
```

Comment:
Use the redirect link: https://www.ni.com/r/ask-ni-community-gweb

Reply:
Okay I will update all of them. Thanks